packages/api/src/controllers/auth.ts

```diff
@@ -72,7 +72,7 @@ app.all(
     res.header("x-livepeer-project-id", req.project?.id || "");
     res.header(
       "x-livepeer-is-project-default",
-      (req.project?.id == req.user?.defaultProjectId).toString(),
+      req.project?.id == req.user?.defaultProjectId ? "true" : "false",
```

Comment:
Given we have the `?` in the `project`, which implies that it could be undefined, WDYT of being resilient to that?

```suggestion
      !req.project?.id || req.project.id == req.user.defaultProjectId ? "true" : "false",
```
(also `req.user` is **never** undefined if there's an auth middleware)

On the bigger picture, all the `?` on projects solved the case when there was no default project for a user, which is not possible anymore AFAIK? Either way, I think we should keep the code consistent and we have a lot of other places that are considering the `req.project` nullable so might as well do it here. Later we can do a little clean-up removing all this "optional" logic which can get confusing for readers in the future.